src/code/stores/simulation-store.coffee

```diff
@@ -115,10 +121,19 @@ SimulationStore   = Reflux.createStore
         speed: @settings.speed
         capNodeValues: @settings.capNodeValues
 
+        # Simulation events get triggered as Actions here, and are
+        # available to anyone who listens to this store
         onFrames: (frames) =>
           SimulationActions.simulationFramesCreated(frames)
           if @settings.isRecording
-            SimulationActions.recordingFramesCreated(frames)
+            if @settings.isRecordingPeriod
+              SimulationActions.recordingFramesCreated(frames)
+            else
+              # Strip out the time information if we are simulating without time...
```

Comment:
Yes, it doesn't seem like that has made it into the current API yet. (I saw that its committed -- but not deployed)